riemann-java-client/src/main/java/com/aphyr/riemann/client/TcpHandler.java

```diff
@@ -1,26 +1,9 @@
 package com.aphyr.riemann.client;
 
-import java.net.InetSocketAddress;
```

Comment:
Yes, this is my pre-commit hock in action :) I'm not sure If I tried a build after these changes, though I generally trust intellij on this.
I'll verify & let you know